### PR TITLE
New gallery identity

### LINKED ISSUE #5087: Update Gallery app to use Flutter's new identity assets
We have a new color scheme, logo, and background image. We should have everything we need to update the Gallery.

Assets are in a Google Drive folder now, but we intend to add everything to a github repo and license them.

### LINKED ISSUE #5147: Gallery Drawer header should be pretty
Right now it just says "Flutter gallery" (not even the right caps) and no images.  @dragostis made a fancier drawer header.  Maybe we can show the Flutter logo here a least?

### PR DESCRIPTION
**Pull request by HansMuller**
- Theme is now lightBlue
- Updated appbar background
- Updated Launcher icon
- Reduced the size or many of the image assets
- Updated the about box icon
- Disabled landscape mode

Fixes #5087
Fixes #5147

### CODE REVIEW

**Comment by HansMuller** [comment]
Note: IOS icons and landscape mode configuration is still TBD

File: examples/flutter_gallery/lib/gallery/drawer.dart [hunk 1]
```diff
@@ -49,7 +49,15 @@ class GalleryDrawer extends StatelessWidget {
       child: new Block(
         children: <Widget>[
           new DrawerHeader(
-            child: new Center(child: new Text('Flutter gallery'))
+            child: new Center(
+              child: new Padding(
+                padding: const EdgeInsets.all(16.0),
+                child: new Image.asset(
+                 'packages/flutter_gallery_assets/drawer_logo.png',
+                  fit: ImageFit.contain
```
Comment (abarth) [inline examples/flutter_gallery/lib/gallery/drawer.dart]: @Hixie Should we use our fancy logo decoration here?

**Comment by abarth** [comment]
LGTM

File: examples/flutter_gallery/lib/gallery/drawer.dart (hunk 1 above)
Comment (Hixie) [inline examples/flutter_gallery/lib/gallery/drawer.dart]: Let me finish it first and then we'll discuss it. :-)

**Pull request merged**